src/Umbraco.Cms.Integrations.Search.Algolia/Controllers/SearchController.cs

```diff
@@ -122,18 +128,27 @@ public async Task<IActionResult> BuildIndex([FromBody] IndexConfiguration indexC
 
                 foreach (var contentDataItem in indexContentData)
                 {
+                    var sw = new Stopwatch();
```

Comment:
I will be removing the stopwatch before releasing the new version, as I think it fits better for testing purposes.